### PR TITLE
auto blur input after change

### PR DESCRIPTION
**Pull request by zhoufosho**

### CODE REVIEW

**Comment by zhoufosho** [comment]
https://app.asana.com/0/25803507664280/37514908904655

**Comment by zhoufosho** [comment]
@jerrysu @TerranceN @dustinhorton

File: src/Select.js [hunk 1]
```diff
@@ -284,6 +290,7 @@ var Select = React.createClass({
 
 	selectValue: function(value) {
 		if (!this.props.multi) {
+			this._blurAfterUpdate = (value != 0);
```
Comment (TerranceN) [inline src/Select.js]: Why `!= 0`? I thought all the values were strings (at least we get back a comma separated list of strings when we listen to onChange)

**Comment by zhoufosho** [comment]
Will be addressed by another PR

**Pull request closed without merging**